support/k8s/k8s-workload-registrar/mode-crd/controllers/pod_controller.go

```diff
@@ -205,6 +206,14 @@ func (r *PodReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 // Reconcile creates a new SPIFFE ID when pods are created
 func (r *PodReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+
+	match1, _ := regexp.MatchString(`^kube-.*`, req.NamespacedName.Namespace)
+	match2, _ := regexp.MatchString(`^openshift.*`, req.NamespacedName.Namespace)
+
+	if match1 == true || match2 == true {
+		return ctrl.Result{}, nil
+	}
+
```

Comment:
Seems like you're just checking for those two static namespaces, `kube-.*` and `openshift-.*`. What the issue is looking for is allowing the user to specify any number of regexes for namespaces to disable. Take a look at [containsString()](https://github.com/spiffe/spire/blob/b010348ac4e31d838eef93cb3ecfedc6584402a5/support/k8s/k8s-workload-registrar/mode-crd/controllers/utils.go#L132-L139)  called just below which currently does the filtering. It just statically checks if the strings are equal. Try making a change in that function to do a regex match.